README.md

```diff
@@ -115,7 +115,7 @@ The easiest way to try this, is by running a cluster with the CockroachDB Operat
 
 ```bash
 kubectl scale cockroachdbs basic --replicas 5 # scaling up from 3
-kubectl scale cockroachdbs basic --replicas 3 # scaling down from 3 (decommission of pod 3 and 4)
+kubectl scale cockroachdbs basic --replicas 3 # scaling down from 5 (decommission of pod 3 and 4)
```

Comment:
That's actually correct the way it is. In the first command we scale up to 5 and in this command we scale down from 5. :blush:

Reply:
Hm? That's why I changed the the command to `down from 5`.

Reply:
true that. Nevermind! Good catch!